public/src/shared/footer.js

```diff
@@ -1,9 +1,137 @@
-function Footer() {
-    return (
-        <div className="Footer">
+import React from 'react';
+import {useState, useEffect} from 'react';
+import Newsletter from './newsletter';
+import { AiFillFacebook } from 'react-icons/ai';
+//see documentation: https://react-icons.github.io/react-icons/
+import { AiOutlineInstagram } from 'react-icons/ai';
+import { AiOutlineCopyright } from 'react-icons/ai';
+
+let footerVars = {
+    // nav: [
+    //     {
+    //         name: 'Home',
+    //         link: '#',
+    //     },
+    //     {
+    //         name: 'Store',
+    //         link: '#',
+    //     },
+    //     {
+    //         name: 'F.A.Q.',
+    //         link: '#',
+    //     },
+    //     {
+    //         name: 'Our Facilities',
+    //         link: '#'
+    //     },
+    //     {
+    //         name: 'Create Account',
+    //         link: '#',
+    //     },
+    //     {
+    //         name: 'Login',
+    //         link: '#',
+    //     },
+    // ],
+    socialMedia: [
+        {
+            name: (<AiFillFacebook className="footer-icon"/>),
+            link: '#',
+        },
+        {
+            name: (<AiOutlineInstagram className="footer-icon"/>),
+            link: '#',
+        },
+    ],
+};
+
+const Footer = () => {
+    const [isDesktop, setDesktop] = useState(window.innerWidth > 650);
+  
+    const updateMedia = () => {
+      setDesktop(window.innerWidth > 650);
+    };
+  
+    useEffect(() => {
+      window.addEventListener("resize", updateMedia);
+      return () => window.removeEventListener("resize", updateMedia);
+    });
 
+    //I'm leaving some unused code here in case our client decides he wants to have nav links in the footer.
+    const itemMap = (arr, target) => {
+        if (target) {
+            return arr.map(e => <a href={e.link} rel="noopener noreferrer" target='_blank'>{e.name}</a>)
+        }
+        return arr.map((e, i) => {
+            if (i === 0) {
+                return <a href={e.link}>{e.name}</a>;
+            } else { return <a href={e.link}> • {e.name}</a>};
+        })
+    }
+
+    const desktopFooter = (
+        <div id='footer' className='container-fluid'>
```

Comment:
We could probably just make sure Peter is okay with this

Reply:
yeah i wanted to talk to the client and see if they were married to the old style. because i think this style works just fine and it would save me work if i could just use it